config/software/ruby.rb

```diff
@@ -127,6 +127,17 @@
     # be fixed.
   end
 
+  # Fix reserve stack segmentation fault when building on RHEL5 or below
+  # Currently only affects 2.1.7 and 2.2.3. This patch taken from the fix
+  # in Ruby trunk and expected to be included in future point releases.
+  # https://redmine.ruby-lang.org/issues/11602
+  if ohai['platform_family'] == 'rhel'  &&
```

Comment:
I'm getting a variety of `NameError`s trying to use sugar within `omnibus-supermarket` which is pinned to a pretty old version of `omnibus`. I'm inclined to keep the longer, but working direct calls to `ohai`.